packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -121,11 +116,15 @@ export class OrgList implements vscode.Disposable {
   }
 
   public async updateOrgList() {
-    const authInfoObjects = await this.getAuthInfoObjects();
-    if (!authInfoObjects) {
+    const orgAuthorizations:
+      | OrgAuthorization[]
```

Comment:
what does that leading `|` do again... is it like a dangling comma for easier diffs?

Reply:
Actually this isn't needed.  I can remove the explicit typing since it is inferred.  Also updating the `if (!orgAuthorizations)` to use .length since an array is always returned here.

Reply:
Nit pick (soap box): I know Prettier is changing the code when autosave and converts the code to that, but lines 119-121 are pretty unreadable to me.  I still don't know what it does.  I'd ever prefer writing this out long hand than what's on 119-121.

Reply:
This block has changed - let me know if you have any comments on the latest!